dali/plugin/plugin_manager_test.cc

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include "dali/plugin/plugin_manager.h"
+
+#define DUMMY_PLUGIN_LIB TEST_PLUGINS_BINARY_DIR "/dummy/libcustomdummyplugin.so"
```

Comment:
I think we should put the dummy plugin in the same output directory as the dali_test.bin and then I think linker will find it with relative path. If not, then perhaps we could try to find test executable's binary using `readlink("/proc/self/exe", ...)` and extract directory from there.